.eslintrc.js

```diff
@@ -33,6 +33,8 @@ module.exports = {
   },
   rules: {
     '@typescript-eslint/no-empty-function': 'off',
+    // https://github.com/vuejs/language-tools/issues/ 47#issuecomment-765562095
+    '@typescript-eslint/no-unused-vars': 'off',
```

Comment:
ESLint complains about unused vars when using `script setup`. Instead, we'll leave it to TypeScript to warn us about unused vars.